manager/celery_.py

```diff
@@ -112,7 +112,7 @@ def battle(name, user, num_times, config):
     try:
         agents = "test::a.pommerman.agents.SimpleAgent,test::a.pommerman.agents.SimpleAgent,test::a.pommerman.agents.SimpleAgent".split(",")
         agent_id = random.randint(0, len(agents))
-        agents.insert(agent_id, "docker::%s/%s" % (user, name))
+        agents.insert(agent_id, "docker-agent::%s/%s" % (user, name))
```

Comment:
why add the "-agent" here?

Reply:
That was bad refactor. Thanks for catching it!